### PR TITLE
mopidy: update to 3.0.1

### PR DESCRIPTION
**Pull request by Eluminae**

### CODE REVIEW

**Comment by Eluminae** [comment]
Changes based on aur changesets https://git.archlinux.org/svntogit/community.git/commit/?h=packages/mopidy&id=fdf23cde00ec4a0436b97ecfef2460a4ca31bed1

**Comment by Eluminae** [comment]
Traceback (most recent call last):
  File "/bin/mopidy", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3251, in <module>
    def _initialize_master_working_set():
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3234, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3263, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 786, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'Pykka>=2.0.1' distribution was not found and is required by Mopidy

**Comment by Eluminae** [comment]
The error was fixed updating the python-pykka package to 2.0.2. Mopidy 3.0.1 works ! Youhou

File: srcpkgs/mopidy/template [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-revision=2
+version=3.0.1
+revision=1
 archs=noarch
 wrksrc="Mopidy-${version}"
-build_style=python2-module
+build_style=python3-module
 pycompile_module=mopidy
-hostmakedepends="python-setuptools python-Sphinx python-sphinx_rtd_theme python-pykka"
+hostmakedepends="python3-setuptools python3-Sphinx python3-sphinx_rtd_theme python-pykka"
```
Comment (ndowens) [inline srcpkgs/mopidy/template]: python-pykka -> python3-pykka

File: srcpkgs/mopidy/template [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 depends="gst1-python gst-plugins-good1 gst-plugins-ugly1 gst-plugins-bad1
- python-futures python-tornado python-pykka python-setuptools python-requests"
+ python3-tornado python-pykka python3-setuptools python3-requests"
```
Comment (ndowens) [inline srcpkgs/mopidy/template]: python-pykka -> python3-pykka

File: srcpkgs/python-pykka/template [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
-hostmakedepends="python-setuptools"
+hostmakedepends="python-setuptools python3-setuptools"
 depends="python"
 short_desc="Python implementation of the actor model"
 maintainer="allan <mail@may.mooo.com>"
 homepage="https://pypi.org/project/Pykka/"
 license="Apache-2.0"
 distfiles="${PYPI_SITE}/P/Pykka/Pykka-${version}.tar.gz"
-checksum=e847ffeadee49b563426ab803e8eee67264d773e38ca14763fdcda56411e3c11
+checksum=895cc2ed8779b65dd14a90ba3f4b8cb0f7904c7bf0710fe96a923019f8e82a39
```
Comment (ndowens) [inline srcpkgs/python-pykka/template]: You are missing
python3-pykka_package() 
.....
and the ln -s srcpkgs/python-pykka srcpkgs/python3-pykka

File: srcpkgs/mopidy/template [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
 pkgname=mopidy
-version=2.3.1
-revision=2
+version=3.0.1
+revision=1
 archs=noarch
 wrksrc="Mopidy-${version}"
-build_style=python2-module
+build_style=python3-module
 pycompile_module=mopidy
```
Comment (ndowens) [inline srcpkgs/mopidy/template]: No need anymore

File: srcpkgs/mopidy/template (hunk 1 above)
Comment (Eluminae) [reply srcpkgs/mopidy/template]: In fact, python-pykka is a python2 and python3 compatible package. There is no python3-pykka cause the python-pykka one is enough

File: srcpkgs/mopidy/template (hunk 4 above)
Comment (Eluminae) [reply srcpkgs/mopidy/template]: Yes !

Comment (ndowens) [reply srcpkgs/mopidy/template]: > Yes !

I've been told fairly recently, when updating python packages to remove them

File: srcpkgs/python-pykka/template (hunk 3 above)
Comment (Eluminae) [reply srcpkgs/python-pykka/template]: Oh ok ! I'm on it

File: srcpkgs/mopidy/template (hunk 1 above)
Comment (ndowens) [reply srcpkgs/mopidy/template]: We split python2 and python3, in which the comment about missing the python3-pykka_package() part does, and reason also reason the need to change the dependency from python-pykka to python3-pykka.

Comment (Eluminae) [reply srcpkgs/mopidy/template]: Understood

File: srcpkgs/python-pykka/template (hunk 3 above)
Comment (Eluminae) [reply srcpkgs/python-pykka/template]: I missed the trick. My bad!

File: srcpkgs/mopidy/template (hunk 4 above)
Comment (Eluminae) [reply srcpkgs/mopidy/template]: Removed. Where do you told things ?

Comment (ndowens) [reply srcpkgs/mopidy/template]: https://github.com/void-linux/void-packages/pull/17926 3rd resolved is where I was told

Comment (Eluminae) [reply srcpkgs/mopidy/template]: Do you want me to remove + unindex python2-pykka ?

Comment (ndowens) [reply srcpkgs/mopidy/template]: Not sure what you are "pointing" to. I am confused because of + unindex

Comment (Eluminae) [reply srcpkgs/mopidy/template]: I was learning about your way to mark packages as "deleted" using the removed-packages replaces. I guess the "+ unindex" actually is an automatic step executed by xbps.

So, do you prefer me to put python-pykka in the removed package replace list? It seems to be used only by mopidy atm

Comment (ndowens) [reply srcpkgs/mopidy/template]: Not sure how committers want to approach this currently, as I was changing them but it has been a bit of a argument ATM, so currently just leave it a mix until someone who can commit looks over this.

Comment (Eluminae) [reply srcpkgs/mopidy/template]: Okay! Ty for your review. I mark conversation as resolved :)

**Pull request merged**